transaction-history-manager/build.gradle.kts

```diff
@@ -3,6 +3,18 @@ plugins {
 }
 
 dependencies {
```

Comment:
several dependencies are already defined in plugins, check them and remove duplicate